<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CEnvironmentSettingsWidget</class>
 <widget class="QWidget" name="CEnvironmentSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="labelGeneral">
     <property name="text">
      <string>General</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidgetSystem">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabSystem">
      <attribute name="title">
       <string>System</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QCheckBox" name="checkBoxLoadPreviousProjectOnStartup">
         <property name="text">
          <string>Show previous project dialog on startup</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>178</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="checkBoxStartFullScreen">
         <property name="text">
          <string>Start fullscreen</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabRServer">
      <attribute name="title">
       <string>R Server</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditHostnameAndPort"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="labelURL">
         <property name="text">
          <string>Hostname:port</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QCheckBox" name="checkBoxShowMessageOnError">
         <property name="text">
          <string>Show message dialog on error</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
